mscore/musescore.cpp

```diff
@@ -1087,6 +1089,7 @@ MuseScore::MuseScore()
       menuFormat->addSeparator();
 
       menuFormat->addAction(getAction("reset-beammode"));
+      menuFormat->addAction(getAction("reset-groupings"));
```

Comment:
I was considering that too. I ended up putting it with the option to reset the beam mode because they are both related to grouping around a beat, but perhaps you are right and this should be considered a "content change" rather than a "formatting change". I will think about it after GSoC.